### ISSUE TITLE
update for svelte 4

### ISSUE THREAD

**Issue by ptone** [jacobbowdoin/sveltefirets#14]
Might be as simple as the dep in package.json 

https://svelte.dev/blog/svelte-4#migrating

**Comment by binary-person** [comment]
for anyone looking for a simple temporary fix without `--force` or `--legacy-peer-deps` add the following to the `package.json`
```
  "overrides": {
    "sveltefirets": {
      "svelte": "^3.0.0 || ^4.0.0"
    }
  }
```

**Comment by jacob-8** [comment]
Sorry to not update. I still use Firebase but for all new work moving forward I'm using Supabase so don't expect new features here. But I'm more than happy to merge PRs and would love someone to submit a PR for Svelte 4-5 support. Should be pretty easy.

**Comment by jacob-8** [comment]
Fixed in `v0.0.38`
